Update Localization Packs page (#900)
PrestaShop/docs: src/content/1.7/development/internationalization/_index.md

# Internationalization

PrestaShop has been designed to be adaptable to any country, region, language or culture.

To achieve this goal, PrestaShop relies on multiple features:

1. There is a Translator system that you can use to translate pre-defined strings. For example, using the translation you can print "Add to cart" in multiple languages in your shop.

2. This Translator can load multiple translation sources, it can be the software translation catalog files, it can be modules catalog files, it can be theme catalog files...

> 3. For each translation catalog files, PrestaShop provides pre-translated packs for [a lot of languages](https://crowdin.com/project/prestashop-official).

Reviewer: ```suggestion
3. For each translation catalog file, PrestaShop provides pre-translated packs for [a lot of languages](https://crowdin.com/project/prestashop-official).
```

# Internationalization

PrestaShop has been designed to be adaptable to any country, region, language or culture.

> To achieve this goal, PrestaShop relies on multiple features.

Reviewer 2: ```suggestion
To achieve this goal, PrestaShop relies on multiple features:
```

> 1. There is a Translator system that you can use to translate pre-defined strings. For example using the translation you can print "Add to cart" in multiple languages in your shop.

Reviewer 2: ```suggestion
1. There is a Translator system that you can use to translate pre-defined strings. For example, using the translation you can print "Add to cart" in multiple languages in your shop.
```

2. This Translator can load multiple translation sources, it can be the software translation catalog files, it can be modules catalog files, it can be theme catalog files...

> 3. For each translation catalog files, PrestaShop provides pre-translated packs for [80 languages](https://crowdin.com/project/prestashop-official).

Reviewer 2: ```suggestion
3. For each translation catalog files, PrestaShop provides pre-translated packs for [a lot of languages](https://crowdin.com/project/prestashop-official).
```
